lib/string.gd

```diff
@@ -939,3 +939,34 @@ DeclareGlobalFunction("StringOfMemoryAmount");
 DeclareGlobalFunction("StringFormatted");
 DeclareGlobalFunction("PrintFormatted");
 DeclareGlobalFunction("PrintToFormatted");
+
+
+#############################################################################
+##
+##  <#GAPDoc Label="Pluralize">
+##  <ManSection>
+##  <Func Name="Pluralize" Arg='count, string[, plural]'/>
+##
+##  <Description>
+##    TODO.
```

Comment:
I'd approve once this is filled in :-)

Reply:
I'll write some tests in a test file, too :-)